### DISCUSSION TITLE
2022/09/27 Meeting Minutes

### DISCUSSION THREAD

**Discussion by aaronhsli** [amosyu2000/cyclops#35] [Meeting Minutes]
## Housekeeping
- Book a room for the next meeting [here](https://library-bookings.mcmaster.ca/Web/schedule.php?sid=14).

### Next meeting
Date: Next Tuesday  
Time: 2:30PM - 3:30PM  
Room: 62 Emerson Living Room

## Stand up
N/A

## Discussion Topics
SRS Document (Due Oct. 6)
- Reference Material (Aaron Li)
- Introduction (Amos Cheung)
- General System Description (Manny Lemos)
- Specific System Description (Amos Yu)
- Functional Requirement (Brian Le, Manny Lemos)
    - Formal Requirements (FSM, Finite State Table), 4 Variable Model
- Non-Functional Requirement (Aaron Li)
- Changes (Amos Cheung)

Manny will make sure we cover everything in the slide. 
- Identify Stakeholders
- Consider use cases
- Identify inputs, outputs
- What you want, not how to do it

PR Request (Issue Numbers, etc.)
- One Commit per PR or Multiple Commits per PR
   - Multiple Commits is okay, not multiple README commits though

Equipment
- Two cameras - one for front-facing, other one for blind spots
-  Sensors? Chassis? Computer Vision to detect both sides. 
- Mounting the holder on the handlebars
- AI to determine whether or not it's a car vs something else
   - Measuring relative velocity
      - Use speedometer detect if biker is moving
      - Use CV to detect if car is getting closer
- Getting more Raspberry PI
- We want a larger POV to see behind cyclist
- Basic blindspot detection is our main goal; Detecting high-velocity vehicles would be stretch goal

**Comment by amosyu2000** [comment]
![image](https://user-images.githubusercontent.com/46848538/192619241-82cc8e39-3998-4a38-8257-8498efafc848.png)
![image](https://user-images.githubusercontent.com/46848538/192619274-98ef3655-4034-4799-b1ad-69af76d963e2.png)
![image](https://user-images.githubusercontent.com/46848538/192619307-ced0d4b4-3c41-4033-8960-bb5444fe05cd.png)

**Comment by amosyu2000** [comment]
@aaronhsli 
![image](https://user-images.githubusercontent.com/46848538/192620690-91ae523b-b8e7-4dad-bd47-487d9e6c3f2b.png)
![image](https://user-images.githubusercontent.com/46848538/192620739-3cf4ff81-5923-48b6-86da-374aebc3d9eb.png)
